### ISSUE TITLE
Trendvis overrides default matplotlib figure size.

### ISSUE THREAD

**Issue by jsegu** [matplotlib/trendvis#26]
Congrats and thank you for this beautiful library! I gave it a try and take the liberty to report here two small issues (though being far from critical).

Trendvis currently use a default figure size of 10 by 10 inches. I wonder if it would be possible to replace this value by the matplotlib default stored in rcParams? This would allow each user to set its own defaults in a matplotlibrc file.

```python
>>> import matplotlib.pyplot as plt
>>> import trendvis
>>> plt.rcParams['figure.figsize'] = '3, 4'
>>> grid = trendvis.XGrid([1,2,1])
>>> print grid.fig.get_size_inches()
[ 10.  10.]
```

In my opinion, the make_grid wrapper would ideally use the rc value too, so that figsize can become an optional argument.

**Comment by mscross** [comment]
This is a great idea- want to make a PR, @jsegu ?

**Comment by mscross** [comment]
Closed by #28
